app/services/region_backfill.rb

```diff
@@ -85,9 +85,9 @@ def create_region_types
     unless dry_run?
       root = RegionType.create! name: "Root", path: "Root"
       org = RegionType.create! name: "Organization", parent: root
-      facility_group = RegionType.create! name: "FacilityGroup", parent: org
-      block = RegionType.create! name: "Block", parent: facility_group
-      _facility = RegionType.create! name: "Facility", parent: block
+      facility_group = RegionType.create! name: "District", parent: org
+      zone = RegionType.create! name: "Zone", parent: facility_group
```

Comment:
I think we should stick with Block here to keep a [ubiquitous language](https://www.martinfowler.com/bliki/UbiquitousLanguage.html) with our primary deployment.